src/Components/CreateImageWizardV2/steps/ImageOutput/Environment.tsx

```diff
@@ -0,0 +1,405 @@
+import React, { Dispatch, SetStateAction } from 'react';
+
+import {
+  Checkbox,
+  FormGroup,
+  Popover,
+  Radio,
+  Text,
+  TextContent,
+  TextVariants,
+  Tile,
+} from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+
+import {
+  useGetArchitecturesQuery,
+  Distributions,
+} from '../../../../store/imageBuilderApi';
+import { provisioningApi } from '../../../../store/provisioningApi';
+import { useGetEnvironment } from '../../../../Utilities/useGetEnvironment';
+
+type useGetAllowedTargetsPropType = {
+  architecture: 'x86_64' | 'aarch64';
```

Comment:
When imported, we can use an existing type. That way if new architectures are added in the future we won't have to update the enums manually throughout the code :sunglasses: 

```suggestion
  architecture: ArchitectureItem['arch'];
```